component/service/src/main/java/org/exoplatform/social/rest/impl/activity/ActivityRestResourcesV1.java

```diff
@@ -148,7 +150,8 @@ public Response getActivities(
                                     required = false
                                 )
                                 @QueryParam("expand")
-                                String expand) {
+                                String expand,
+                                @ApiParam(value = "activityStreamType enum name", required = false) @QueryParam("streamType") ActivityStreamType streamType) {
```

Comment:
> activityStreamType enum name

It would be better to add more significant description of the parameter, this will make it possible to use this API without having to guess/read the source code. In fact, with those description, we would like to generate a website for REST API documentation like https://docs.cloud.coinbase.com/sign-in-with-coinbase/docs/api-users?javascript#show-a-transaction or https://developers.google.com/calendar/api/v3/reference/events

For example : "Activity stream type. Possible values: ALL, USER_STREAM, SPACE_STREAM, FAVORITE_STREAM, MANAGED_SPACES_STREAM, FAVORITE_SPACES_STREAM. Default: ALL."